RELEASE-NOTES.md

```diff
@@ -1,5 +1,44 @@
 # Prefect Release Notes
 
+## Release 2.8.5
+
+### Enhancements
+- Add basic event instrumentation to blocks — https://github.com/PrefectHQ/prefect/pull/8686
+- Add an endpoint to retrieve data from the collection registry — https://github.com/PrefectHQ/prefect/pull/8685
+
+### Fixes
+- Add carriage return to `prompt_select_from_list` — https://github.com/PrefectHQ/prefect/pull/8705
+- Fix Kubernetes job watch timeout request error by rounding — https://github.com/PrefectHQ/prefect/pull/8733
+- Ignore fsspec `2023.3.0` during requirements installation — https://github.com/PrefectHQ/prefect/pull/8757
+- Use `PREFECT_API_URL` when initializing the events client — https://github.com/PrefectHQ/prefect/pull/8704
+- Fix Deployment and Concurrency Limit pages tabs — https://github.com/PrefectHQ/prefect/pull/8716
+- Add tests for base exceptions and calls — https://github.com/PrefectHQ/prefect/pull/8734
+- Remove deployment flow run foreign key — https://github.com/PrefectHQ/prefect/pull/8684
```

Comment:
Arguably an enhancement not a fix; what's the user-facing effect? Improve deletion of deployments?

Reply:
@rpeden are we going to include more details about the ux here?

Reply:
yes, added in latest commit